packages/nextclade-cli/src/io/http_client.rs

```diff
@@ -34,6 +35,13 @@ pub struct ProxyConfig {
   #[clap(long)]
   #[clap(value_hint = ValueHint::Other)]
   pub proxy_pass: Option<String>,
+
+  /// Path to extra CA certificates
+  ///
+  /// You can also provide path to CA certificates in environment variable `NEXTCLADE_EXTRA_CA_CERTS`. The argument takes precedence over environment variable if both are provided.
+  #[clap(long)]
+  #[clap(value_hint = ValueHint::Other)]
+  pub extra_ca_certs: Option<PathBuf>,
```

Comment:
This is in `ProxyConfig`, but it's not really a proxy configuration item. It may be used often with proxies, but it won't _only_ be used with proxies.

Reply:
I think `--extra-ca-certs` is the right name given `NEXTCLADE_EXTRA_CA_CERTS`. We could switch from "CA" to "root" if we wanted, but "CA certs" is a pretty common term-of-art.

Reply:
(I wouldn't say just `--ca-certs` because that implies replacing _all_ the default CA certs too and is how that option is used by other software.)